includes/user-functions.php

```diff
@@ -549,7 +549,7 @@ function edd_get_customer_address( $user_id = 0 ) {
 		'state'   => '',
 	) );
 
-	return $address;
+	return apply_filters( 'edd_get_customer_address', $address, $user_id );
```

Comment:
This new filter should be documented: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters